modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -69,3 +76,25 @@ Components are transitioning to [Compound Components](./COMPOUND_COMPONENTS.md)
 PR:
 
 - https://github.com/Workday/canvas-kit/pull/953
+
+### Button
+
+- `beta_Button` export was removed.
+- 🤖 The default export was removed. The codemod will change default imports to named imports.
+- 🤖 Enums have been removed from all buttons. String literals are used instead. The codemod will write any usages of an enum to the string literal. If you used an enum as a type, the codemod will expand to a union of string literals. You could change the union manually instead to be something like `ButtonProps['variant']` if you prefer not to duplicate the union of string literals.
+  ```tsx
+  // before
+  <Button variant={Button.Variant.Primary} />
+  interface Props {
+    variant: ButtonVariant
+  }
+  // after
+  <Button variant="primary" />
+  interface Props {
+    variant: 'primary' | 'secondary'
+  }
+  ```
+- Buttons now use `createComponent` utility from the `common` module which now forwards `ref` and allows `as` to change the underlying element.
+  - 🤖 `buttonRef` was renamed to `ref` provided by `createComponent`. `buttonRef` is no longer valid. The codemod will rewrite all instances of `buttonRef` to `ref`.
```

Comment:
May be worth a warning/note/blockquote to call out that this could affect other code unintentionally.